src/bot.ts

```diff
@@ -392,23 +392,24 @@ a known bot info object.",
      */
     async start(options?: PollingOptions) {
         // Perform setup
+        const setup: Promise<void>[] = [];
         if (!this.isInited()) {
-            await this.init(this.pollingAbortController?.signal);
+            setup.push(this.init(this.pollingAbortController?.signal));
         }
         if (this.pollingRunning) {
+            await Promise.all(setup);
```

Comment:
Is this line reached with `setup` not being empty?

Reply:
Yes, if the bot is used in a weird way. The bot info field can be cleared.